MFE/common/src/components/WizardForms/TemplateCreation/TemplateCreation.jsx

```diff
@@ -39,6 +39,8 @@ const TemplateCreation = ({
   const { t } = useTranslation(['templateCreation', 'attrs']);
   const classes = useStyles();
 
+  const [attrLabelError, setAttrLabelError] = useState(false);
```

Comment:
There is just a boolean to validate ALL attrs. If I add 3 attrs and type an invalid label on the last one, the error will appear in all label fields?